docs/content/docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -79,7 +79,7 @@ Congratulations! You have successfully run a Flink application by deploying Flin
 
 ## Deployment Modes
 
-For production use, we recommend deploying Flink Applications in the [Application Mode]({{< ref "docs/deployment/overview" >}}#application-mode), as these modes provide a better isolation for the Applications.
```

Comment:
maybe this change can go into another commit(it is not related to the translation)